background/sync-manager.js

```diff
@@ -63,19 +63,19 @@ const syncMan = (() => {
       return status;
     },
 
-    async login(name = prefs.get('sync.enabled')) {
+    async login(name) {
       if (ready.then) await ready;
+      if (!name) name = prefs.get('sync.enabled');
+      await tokenMan.revokeToken(name);
```

Comment:
What if this call fails?

Reply:
Look at the code of revokeToken: if it fails then something is so wrong that catching it won't change a thing.

Reply:
`revokeToken` itself ignores fetch errors. If other services fail e.g. `storage.remove` then the user won't be able to login.